src/main/java/org/alfresco/repo/jscript/ScriptNode.java

```diff
@@ -2706,7 +2706,10 @@ public ScriptNode cancelCheckout()
      * @param mimetype   Mimetype destination for the transformation
      * 
      * @return Node representing the newly transformed document.
+     *
+     * @deprecated The transformations code is being moved out of the codebase and replaced by the new async RenditionService2 or other external libraries.
      */
+    @Deprecated
     public ScriptNode transformDocument(String mimetype)

```

Comment:
This functionality will be preserved and the implementation will change.
This ScriptNode is a public API and we have to preserve the features it provides.